node/tests/GlideClusterClient.test.ts

```diff
@@ -998,17 +1000,6 @@ describe("GlideClusterClient", () => {
                         },
                         TIMEOUT,
                     );
-                },
-            );
-        },
-    );
-
-    describe.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
wait, why are we removing this?

Reply:
We were closing and re-opening `describe` blocks, so I just merged them.
Instead
```
describe([RESP2, RESP3] ... it(<test 1>))
describe([RESP2, RESP3] ... it(<test 2>))
```
I made
```
describe([RESP2, RESP3] ... it(<test 1>) it(<test 2>))
```